aws-sdk-v2/build.gradle

```diff
@@ -24,6 +25,7 @@ dependencies {
     testImplementation "software.amazon.awssdk:s3"
     testImplementation "software.amazon.awssdk:dynamodb"
     testImplementation "software.amazon.awssdk:ses"
+    testImplementation 'software.amazon.awssdk:secretsmanager'
```

Comment:
to stay consistent you should use double quotes  here instead of simple quotes

Reply:
Done